### ISSUE TITLE
Installation Question

### ISSUE THREAD

**Issue by LoveraSantiago** [caprica/picam-native#1]
Hi Caprica.

I see the update on picam-java and I'm now trying to update do version 2.
I'm not familiar with C so I'm getting trouble trying to install this picam-native.

I run bash pi.sh on my picam-native folder. And I get this result bellow.
"Arquivo ou diretório nao encontrado" means File or Folder not find.

Can you please appoint me into the right direction?
![Capturar1](https://user-images.githubusercontent.com/7741737/55295276-94296a80-53e1-11e9-8904-b0288f78bc0e.PNG)

**Comment by caprica** [comment]
You need to install a JDK if you don't already have one, and depending on which JDK you have installed change the JNI_INCLUDE and JNI_LIB symbols in pi.sh to point to that directory.

On my Pi, it is here:
```
JNI_INCLUDE=/usr/lib/jvm/java-9-openjdk-armhf/include
JNI_LIB=/usr/lib/jvm/java-9-openjdk-armhf/lib
```
With a full release version of picam-2.x you won't need to do any of this, but picam-2.x is still under development so you will have to suffer these problems, sorry!

**Comment by caprica** [comment]
If your Pi software is all up-to-date, you can try this pre-built library at your own risk:

http://capricasoftware.co.uk/downloads/picam/picam.so

**Comment by caprica** [comment]
This temporary situation should now be resolved with the latest picam-2.0.0 release.

Please post any further issues about this at the picam project.
